src/snowflake/connector/network_async.py

```diff
@@ -191,6 +246,27 @@ async def close_async(self) -> None:
         self._idle_sessions.clear()
 
 
+def make_client_session(loop: asyncio.BaseEventLoop) -> aiohttp.ClientSession:
```

Comment:
nit: is this method used somewhere else? wondering if we can remove this method, since there's only one line in it

Reply:
Do you mean close_async? It's longer than it looks; you have to expand the lines in github.

Reply:
oh I mean the `make_client_session` method